.github/workflows/release.yaml

```diff
@@ -372,8 +372,8 @@ jobs:
           path: /home/runner/go/pkg/mod
           key: GOMODCACHE-v2-${{ hashFiles('**/go.mod') }}
       # https://stackoverflow.com/questions/58033366/how-to-get-current-branch-within-github-actions
-      - run: make changelog VERSION=${GITHUB_REF##*/}
-      - run: cat changelog
+      - run: make release-notes VERSION=${GITHUB_REF##*/}
```

Comment:
it was a mistake to call this `make changelog` because it was not making a changelog, it was making release notes